README.md

```diff
@@ -79,24 +79,11 @@ Install [`npm`](https://www.npmjs.com/get-npm). `npm` is installed and comes bun
 $ brew install node
 ```
 
-Add dependencies for the [Express.js](https://expressjs.com/) server
+Add dependencies for the [Express.js](https://expressjs.com/) server and the [React.js](https://reactjs.org/)/[create-react-app](https://github.com/facebook/create-react-app) client
 
 ```
 $ cd universe-webapp
-$ npm install
-```
-
-Add dependencies for the [React.js](https://reactjs.org/)/[create-react-app](https://github.com/facebook/create-react-app) client
-
-```
-$ cd client
-$ npm install
-```
-
-Go back to the server
-
-```
-$ cd ../
+$ npm install && npm run client-install
```

Comment:
Hey @modkaffes, let's change these changes back as it is easier for beginner devs to understand what's going on. Thx

Reply:
Ah, I saw the `client-install` script in place and thought we could use this for brevity. Should I remove `client-install` as well and just use the old sequence? (cd in, install, cd back)

Reply:
Yeah I thought about having everything run for the developer but I think we should have the developer run the server and then the client manually.

Reply:
Definitely, rolling it back after offline discussion.